bin/MadGraph5_aMCatNLO/cards/production/13TeV/darkphotonDimuon/darkphotonDimuon_DPMass5e-1MeV_DF1e-2/darkphotonDimuon_DPMass5e-1MeV_DF1e-2_customizecards.dat

```diff
@@ -0,0 +1,6 @@
+set param_card HIDDEN 1 5e-4
+set param_card HIDDEN 2 1e4
+set param_card HIDDEN 3 1e-1
+set param_card HIDDEN 4 1e-10
+set param_card DECAY 35 auto
```

Comment:
where do you set the masses of 35 and 1023 ?

Reply:
The hidden parameter 1 is the masss of 1023, hidden parameter 2 is the mass of 35.